app/services/sign_in/token_response_generator.rb

```diff
@@ -74,5 +77,15 @@ def serialized_service_account_token(access_token:)
         }
       }
     end
+
+    def create_user_audit_log(user_verification:)
+      user_action_event = UserActionEvent.find_by(identifier: 'user_login')
+      UserAuditLogger.new(user_action_event:,
+                          acting_user_verification: user_verification,
+                          subject_user_verification: user_verification,
```

Comment:
Given how common it will be for acting == subject, let's allow 'acting' to be blank, but set it equal to 'subject' inside the UserAuditLogger if this is the case